docs/rule/no-arguments-for-html-elements.md

```diff
@@ -0,0 +1,97 @@
+## no-arguments-for-html-elements
```

Comment:
Can you mention how "angle bracket components" use the `@argument` style and that's why users are more likely to start making this mistake?

There's also a rule documentation template FYI: https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/_TEMPLATE_.md